packages/admin/src/models/verification.ts

```diff
@@ -0,0 +1,25 @@
+import { Network } from 'defender-base-client';
+
```

Comment:
Can we add a reference to what file are we copying this from?

Reply:
You mean, adding it to packages/src/utils/base/network.ts?

Reply:
Just a comment, we have some other sections with a `Code copied from /models/.../...`

It has made my day easier some times 😋